themes/src/main/resources/theme/keycloak-preview/account/resources/app/account-service/account.service.ts

```diff
@@ -100,7 +100,12 @@ export class AccountServiceClient {
             this.kcSvc.login();
         }
         console.log(error);
-        ContentAlert.danger(error.name + ': ' + error.message);
+
+        if (error != null && error.response != null && error.response.data != null && error.response.data.errorMessage) {
```

Comment:
Better way to say this:
`if (error?.response?.data?.errorMessage) {`

See https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining

Reply:
Hmm.  Looks like optional chaining is not available until TypeScript 3.7 and we are still on 3.3.  Maybe not such a good idea to upgrade this close to the release.